src/main/resources/application.yml

```diff
@@ -18,15 +18,17 @@ spring:
     username: nakadi
     password: nakadi
     driverClassName: org.postgresql.Driver
-    initialSize: 2
-    maxActive: 2
-    minIdle: 1
-    maxIdle: 1
-    testOnBorrow: true
-    validationQuery: SELECT 1
-    testWhileIdle: true
-    timeBetweenEvictionRunsMillis: 5000
-    minEvictableIdleTimeMillis: 60000
+    tomcat:
+      initial-size: 2
+      max-active: 2
+      min-idle: 1
+      max-idle: 1
+      test-on-borrow: true
+      validation-query: SELECT 1
+      test-while-idle: true
+      time-between-eviction-runs-millis: 5000
+      min-evictable-idle-time-millis: 60000
+      connection-properties: socketTimeout=2;connectTimeout=2;loginTimeout=2
```

Comment:
@adyach unfortunately these timeouts are in seconds, and the type is `int`, so the minimum would be 1 second.